randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -1309,9 +1292,48 @@ Extra - aabb: [122.126015, -44.231895, -102.18276, 201.19897, 105.61425, -30.731
   * Layers: default
   * Pickup 84; Major Location? True
   * Extra - position_required: False
-  > Door to Save Station Mines B
+  > Room Bottom
       Trivial
 
+> Before Fight; Heals? False
+  * Layers: default
+  > Door to Dynamo Access
+      After Invisible Drone
+  > Door to Quarantine Access A
+      All of the following:
+          # The Security Drone event triggers door locks on all doors in Central Dynamo except for the door to Quarantine Access A. However, if Samus leaves this room without picking up the item, the doors to Save Station Mines B and Dynamo Access will then permanently be sealed, making it impossible to backtrack through the room.
```

Comment:
This is redundant with the `Room Bottom -> Door to Quarantine Access A` logic. The player should just be able to go `Door to Dynamo Access -> Room Bottom (Trivial) -> Door to Quarantine Access A`, thus avoiding duplicate logic.